datasets/swedish_ner_corpus/swedish_ner_corpus.py

```diff
@@ -0,0 +1,101 @@
+# coding=utf-8
+# Copyright 2020 HuggingFace Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+import datasets
+
+
+_DESCRIPTION = """\
+Webbnyheter 2012 from Spraakbanken, semi-manually annotated and adapted for CoreNLP Swedish NER. Semi-manually defined in this case as: Bootstrapped from Swedish Gazetters then manually correcte/reviewed by two independent native speaking swedish annotators. No annotator agreement calculated.
+"""
+_HOMEPAGE_URL = "https://github.com/klintan/swedish-ner-corpus"
+_TRAIN_URL = "https://raw.githubusercontent.com/klintan/swedish-ner-corpus/master/train_corpus.txt"
+_TEST_URL = "https://raw.githubusercontent.com/klintan/swedish-ner-corpus/master/test_corpus.txt"
+
+
+class SwedishNERCorpusDataset(datasets.GeneratorBasedBuilder):
```

Comment:
Could you rename it `SwedishNERCorpus` ?
The dataset class must be the camel case equivalent of `swedish_ner_corpus` which is the script name.